timesketch/api/v1/resources/scenarios.py

```diff
@@ -269,3 +270,96 @@ def post(self, sketch_id, scenario_id):
             db_session.commit()
 
         return self.to_json(scenario)
+
+
+class QuestionConclusionListResource(resources.ResourceMixin, Resource):
+    """Resource for investigative question conclusion."""
+
+    @login_required
+    def get(self, sketch_id, question_id):
+        """Handles GET request to the resource.
+
+        Returns:
+            A list of JSON representations of the conclusions.
+        """
+        sketch = Sketch.query.get_with_acl(sketch_id)
+        question = InvestigativeQuestion.query.get(question_id)
+
+        conclusions = InvestigativeQuestionConclusion.filter_by(
+            investigativequestion=question
+        ).all()
+
+        if not sketch:
+            abort(HTTP_STATUS_CODE_NOT_FOUND, "No sketch found with this ID")
+
+        return self.to_json(conclusions)
+
+    @login_required
+    def post(self, sketch_id, question_id):
+        """Handles POST request to the resource.
+
+        Adds or edits a conclusion.
+
+        Returns:
+            A JSON representation of the question.
```

Comment:
In line 239 you return a JSON representation of the conclusion not the question. Please fix the comment here.

Reply:
copy paste error :) Fixed.